src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardConsumer.java

```diff
@@ -388,9 +398,17 @@ ShutdownReason getShutdownReason() {
         return shutdownReason;
     }
 
+    /**
+     * @return the shardSyncer
+     */
+    public ShardSyncer getShardSyncer() {
```

Comment:
Just a tip: KCL uses lombok, and you can create this method by adding the `@Getter` annotation.

```java
@Getter
private final ShardSyncer shardSyncer;
```

Reply:
Using lombok for the getter now.